extensions/rvdatabinding/src/main/java/com/google/android/agera/rvdatabinding/DataBindingRepositoryPresenterCompilerStates.java

```diff
@@ -34,7 +36,8 @@
   /**
    * Compiler state to specify how to bind the {@code View} using data binding.
    */
-  interface DBRPMain<T> extends RPItemCompile<T>, RPSpecificCollectionCompile<T> {
+  interface DBRPMain<T> extends RPItemCompile<T>,
+      RPSpecificCollectionCompile<T>, RPCollectionCompile<T> {
```

Comment:
I didn't know this cheeky interface reuse! maybe make DBRP copies so this outer interface is self-contained.

Also, inline interfaces wherever possible.